tests/conftest.py

```diff
@@ -6,6 +6,27 @@
 from retry import retry
 
 
+@pytest.fixture
+def polldevs_conf(tmp_path):
+    name = tmp_path.joinpath("polldevs.cf")
+    with open(name, "w") as conf:
```

Comment:
Is this the same format as Zino 1?

Reply:
It is. The first code written for Zino 2 was the `polldevs.cf` parser, with 100% coverage.  And voila, suddenly we needed the test data for other tests as well...